test/Serilog.Extensions.Logging.Tests/DisposeTests.cs

```diff
@@ -0,0 +1,72 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Serilog.Core;
+using Serilog.Events;
+using Xunit;
+
+namespace Serilog.Extensions.Logging.Tests;
+
+public class DisposeTests
+{
+    private readonly DisposableSink _sink;
+    private readonly Logger _serilogLogger;
+
+    public DisposeTests()
+    {
+        _sink = new DisposableSink();
+        _serilogLogger = new LoggerConfiguration()
+            .WriteTo.Sink(_sink)
+            .CreateLogger();
+    }
+
+    [Fact]
+    public void AddSerilog_must_dispose_the_provider_when_dispose_is_true()
```

Comment:
The Serilog codebases use `PascalCaseForTestNames()`; shortening this to something like `DisposesProviderWhenDisposeIsTrue()` would help make the scheme work.

Reply:
Sorry, I will address!

Reply:
Naming changed